### PR TITLE
Feature/place migration add fields

### PR DESCRIPTION
**Pull request by ivanshamis**

### CODE REVIEW

File: app/Entities/Place/Place.php [hunk 1]
```diff
@@ -16,6 +16,8 @@
  * @property float  $latitude
```
Comment (nemoipaha) [inline app/Entities/Place/Place.php]: Remove this Class Place docblock

File: database/migrations/2018_08_03_054359_create_places_table.php [hunk 2]
```diff
@@ -20,6 +20,8 @@ public function up()
             $table->float('latitude');
             $table->integer('zip');
             $table->string('address');
+            $table->string('phone');
```
Comment (nemoipaha) [inline database/migrations/2018_08_03_054359_create_places_table.php]: this 2 fields can be nullable

File: app/Entities/Place/Place.php (hunk 1 above)
Comment (ivanshamis) [reply app/Entities/Place/Place.php]: done

File: database/migrations/2018_08_03_054359_create_places_table.php (hunk 2 above)
Comment (ivanshamis) [reply database/migrations/2018_08_03_054359_create_places_table.php]: done

**Comment by ivanshamis** [comment]
Done

**Comment by ivanshamis** [comment]
done

**Pull request merged**